### PR TITLE
Add message for missing Kafka Lib

### PR DESCRIPTION
**Pull request by Azquelt**
When using the Kafka connector for Reactive Messaging, the kafka client jar must be included in the app or in a shared library.

Add a more helpful error message if it is not present.

### CODE REVIEW

**Comment by Azquelt** [comment]
#build

File: dev/com.ibm.ws.microprofile.reactive.messaging.kafka/resources/com/ibm/ws/microprofile/reactive/messaging/kafka/resources/ReactiveMessaging.nlsprops [hunk 1]
```diff
@@ -55,6 +55,11 @@ kafka.groupid.not.set.CWMRX1005E=CWMRX1005E: The group.id property was not set i
 kafka.groupid.not.set.CWMRX1005E.explanation=The Kafka consumer group ID must be set to allow Reactive Messaging to function properly.
 kafka.groupid.not.set.CWMRX1005E.useraction=Set the property in one of the MicroProfile ConfigSource locations for this application, for example, the microprofile-config.properties file.
 
+#Do not translate "kafka-clients", "Apache Kafka" or "Maven Central"
+kafka.library.not.present.CWMRX1006E=CWMRX1006E: Classes from the kafka-clients jar could not be loaded. Ensure that the kafka-clients jar and its dependencies are available as libraries to your application.
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.microprofile.reactive.messaging.kafka/resources/com/ibm/ws/microprofile/reactive/messaging/kafka/resources/ReactiveMessaging.nlsprops]: Change to
"Classes from the kafka-clients JAR file could not be loaded. Ensure that the kafka-clients JAR file and its dependencies are available as libraries to your application."

File: dev/com.ibm.ws.microprofile.reactive.messaging.kafka/resources/com/ibm/ws/microprofile/reactive/messaging/kafka/resources/ReactiveMessaging.nlsprops [hunk 2]
```diff
@@ -55,6 +55,11 @@ kafka.groupid.not.set.CWMRX1005E=CWMRX1005E: The group.id property was not set i
...
+kafka.library.not.present.CWMRX1006E.explanation=Applications which use the Reactive Messaging Kafka Client must provide the Apache Kafka clients jar and its dependencies, either by packaging it as a library within the application, or by providing it as a shared library.
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.microprofile.reactive.messaging.kafka/resources/com/ibm/ws/microprofile/reactive/messaging/kafka/resources/ReactiveMessaging.nlsprops]: Change to
"Applications that use the Reactive Messaging Kafka Client must provide the kafka-clients JAR file and its dependencies. They can provide this file and its dependencies either by packaging it as a library within the application or by providing it as a shared library."

File: dev/com.ibm.ws.microprofile.reactive.messaging.kafka/resources/com/ibm/ws/microprofile/reactive/messaging/kafka/resources/ReactiveMessaging.nlsprops [hunk 3]
```diff
@@ -55,6 +55,11 @@ kafka.groupid.not.set.CWMRX1005E=CWMRX1005E: The group.id property was not set i
...
+kafka.library.not.present.CWMRX1006E.useraction=Package the kafka-clients jar and its dependencies as libraries in your application, or as a shared library. The kafka-clients jar can be downloaded from the Apache Kafka website or from Maven Central.
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.microprofile.reactive.messaging.kafka/resources/com/ibm/ws/microprofile/reactive/messaging/kafka/resources/ReactiveMessaging.nlsprops]: Change to
"Package the kafka-clients JAR file and its dependencies as libraries in your application or as a shared library. The kafka-clients JAR file can be downloaded from the Apache Kafka website or from Maven Central."

**Comment by Azquelt** [comment]
Thanks @Charlotte-Holt, all requested changes have been made.

**Comment by Azquelt** [comment]
#build

**Comment by Charlotte-Holt** [comment]
Thanks, @Azquelt. ID reviewed label added!

**Pull request merged**